idaes/models/properties/general_helmholtz/helmholtz_functions.py

```diff
@@ -1438,14 +1768,17 @@ def build(self):
             "J/mol to J/kg": 1.0 / self.mw,
             "kJ/kg to J/mol": (pyo.units.J * 1000 / pyo.units.kJ) * self.mw,
             "J/mol/K to kJ/kg/K": (pyo.units.kJ / 1000 / pyo.units.J) / self.mw,
-            "J/mol/K to J/kg/K": 1.0 / self.mw,
+            "J/mol/K to J/kg/K": 1 / self.mw,
```

Comment:
I seem to recall @andrewlee94 replacing a bunch of integer zeros or ones with floating point zeros or ones somewhere in core. Is this a place where that distinction matters?

Reply:
I'm pretty sure it doesn't matter here since the division will make it a float anyway.

Reply:
It also doesn't really matter unless you want to look at the expression body itself.